<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>23</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>22</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>56</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="replace">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>53</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_re">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>81</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Regular expression</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_case">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>81</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ignore case</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_wrap">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>81</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wrap around</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_replaceall">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_replace">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_replacefind">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace and find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_prev">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>120</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_next">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>120</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="not_found">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>83</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>not found</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
